src/collective/feedback/restapi/services/configure.zcml

```diff
@@ -8,7 +8,7 @@
   <plone:service
       method="POST"
       factory=".add.FeedbackAdd"
-      for="plone.dexterity.interfaces.IDexterityContent"
+      for="plone.app.layout.navigation.interfaces.INavigationRoot"
```

Comment:
`INavigationRoot` is a `IDexterityContent`, are you sure there are no drawbacks with this change?

Reply:
Yes, I'm sure. This change is breaking, so it requires a specific frontend version to be used.

Reply:
If it is a breaking change, it requires a major version update and an explanation in the PR description and the README.

Reply:
done